todo.sh

```diff
@@ -5,16 +5,16 @@
 # The relative paths to the project files can now  be used
 # like so $lib_dir/someProjectScript.sh
 this_path=${BASH_SOURCE[0]}
-lib_path=$(readlink ${this_path})
-lib_dir=$(dirname $lib_path)
+lib_path="$(readlink "$this_path")"
```

Comment:
Just curious, what are the benefits of using e.g. `"$lib_path"` over `${lib_path}`? Or simply just conventions?

Reply:
I changed this because `$lib_path` was without brackets. I try to only use brackets when they're needed, i.e. `${first}${second}` (because ${firstsecond} is something else) or when doing bash substitutions like `${var/de/12}`.

I'm not sure if there's an **actual** convention for this though, it's nothing like a linter like [shellcheck](https://www.shellcheck.net/) would pick up.